dags/transformation/dbt_nightly.py

```diff
@@ -73,5 +73,25 @@
     dag=dag,
 )
 
+dbt_run_cloud_mattermost_analytics_nightly = KubernetesPodOperator(
+    **pod_defaults,
+    image=MATTERMOST_DATAWAREHOUSE_IMAGE,  # Uses latest build from master
+    task_id="dbt-cloud-mattermost-analytics-nightly",
+    name="dbt-cloud-mattermost-analytics-nightly",
+    secrets=[
+        DBT_CLOUD_API_ACCOUNT_ID,
+        DBT_CLOUD_API_KEY,
+        SNOWFLAKE_ACCOUNT,
+        SNOWFLAKE_USER,
+        SNOWFLAKE_PASSWORD,
+        SNOWFLAKE_TRANSFORM_ROLE,
+        SNOWFLAKE_TRANSFORM_WAREHOUSE,
+        SNOWFLAKE_TRANSFORM_SCHEMA,
+        SSH_KEY,
+    ],
+    env_vars=env_vars,
+    arguments=["python -m  utils.run_dbt_cloud_job 254981 \"Mattermost Analytics DBT nightly\""],
+    dag=dag,
+)
 
-dbt_run_cloud_nightly
+dbt_run_cloud_mattermost_analytics_nightly >> dbt_run_cloud_nightly
```

Comment:
do we need this dependency?
cloud nightly should run irrespective of fail/success of analytics nightly.

Reply:
The reason they run consequently is to spread the load on the database over time. If they run in parallel, there will be a lot of load in some of the already heavily utilized sources. Order can change.

Another approach would be a different DAG with a different schedule. This however wouldn't ensure that the DAGs do not run in parallel.

Reply:
In case the upstream fails, cloud nightly will not be triggered.

Reply:
That's true. Shall I just add a trigger rule on `dbt_run_cloud_nightly` so that it always run?

Reply:
yes, looks good.

Reply:
Adjusted mattermost-analytics to run second, switched tigger rule to `ALL_DONE` (from default `ALL_SUCCESS`).